tensorflow/lite/micro/micro_allocator_test.cc

```diff
@@ -356,7 +352,6 @@ TF_LITE_MICRO_TEST(TestMockModelAllocation) {
 
 TF_LITE_MICRO_TEST(TestMockModelAllocationWithGivenMemoryPlanner) {
```

Comment:
Can we add a test specific for where the issue happens: one model that contains two subgraphs?

Reply:
Done